extension/config/azureFunction/package.json

```diff
@@ -1,11 +1,5 @@
 {
-  "name": "ct-mollie-integration",
-  "version": "1.0.0",
-  "description": "",
   "scripts": {
-    "start": "func start",
-    "test": "echo \"No tests yet...\""
-  },
-  "dependencies": {},
-  "devDependencies": {}
+    "start": "func start"
+  }
```

Comment:
This pakcage.json is required because of the start command [as per commercetools docs](https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-node?tabs=v2-v3-v4-export%2Cv2-v3-v4-done%2Cv2%2Cv2-log-custom-telemetry%2Cv2-accessing-request-and-response%2Cwindows-setting-the-node-version#folder-structure)

Reply:
I don't understand - this is a microsoft link ? Is it from commercetools?

Reply:
sorry, it's indeed azure link, I updated the text